models/models_methods_test.go

```diff
@@ -384,6 +384,83 @@ workflows:
 	}
 }
 
+// Trigger map
```

Comment:
Moved here from trigger_map_test.go, because this tests the trigger map by validating a bitrise.yml content.